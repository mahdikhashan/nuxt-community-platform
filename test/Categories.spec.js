import { mount } from '@vue/test-utils'
import Categories from '@/components/Categories.vue'

describe('Categories', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Categories)
    expect(wrapper.vm).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
