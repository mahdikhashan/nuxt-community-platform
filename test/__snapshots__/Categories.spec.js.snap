// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories is a Vue instance 1`] = `
"<div class=\\"h-[108px] last:border-b-[1px] last:border-gray-300 before:content-[''] before:border-t-[1px] before:block before:border-gray-300 before:w-[100%]\\">
  <div class=\\"flex justify-between items-center h-full w-full relative\\">
    <div class=\\"flex items-center\\">
      <div class=\\"flex flex-col ml-2\\">
        <nuxt-link to=\\"/categories\\" class=\\"font-bold text-lg py-1\\">
          Announcing
        </nuxt-link>
        <nuxt-link to=\\"/categories\\" class=\\"font-light text-sm py-1\\">
          Information about Expensify and using the Community that doesn't relate to the product functionality.
        </nuxt-link>
        <div><span><nuxt-link to=\\"/categories\\"><span></span></nuxt-link> <span class=\\"text-xs inline-block text-gray-500 my-[2px] mx-[4px]\\">12.1K discussions</span> <span class=\\"text-xs inline-block text-gray-500 my-[2px] mx-[4px]\\">5.9K views</span> <span class=\\"text-xs inline-block text-gray-500 my-[2px] mx-[4px]\\">0 comments</span> <span class=\\"text-xs inline-block text-gray-500 my-[2px] mx-[4px]\\">Started by
              <nuxt-link to=\\"/categories\\"><span></span></nuxt-link></span> <span class=\\"text-xs inline-block text-gray-500 my-[2px] mx-[4px]\\">November 2021</span></span></div>
      </div>
    </div>
    <div class=\\"absolute top-2 right-1 flex items-center justify-center gap-1\\">
      <nuxt-link to=\\"/categories\\"><img src=\\"dots.svg\\" alt=\\"booked\\" style=\\"width: 25px;\\"></nuxt-link>
    </div>
  </div>
</div>"
`;
